apps/web/src/data/contract/requests/getDAOMultisig.ts

```diff
@@ -0,0 +1,103 @@
+import axios from 'axios'
+import { Address, checksumAddress, isAddress } from 'viem'
+
+import { AddressType, CHAIN_ID } from 'src/typings'
+
+interface AttestationResponse {
+  data: {
+    attestations: Array<{
+      attester: string
+      recipient: string
+      decodedDataJson: string
+    }>
+  }
+}
+
+interface DecodedData {
+  name: string
+  type: string
+  value: {
+    type: string
+    value: string
+  }
+}
+
+const ATTESTATION_SCHEMA_UID = `0x1289c5f988998891af7416d83820c40ba1c6f5ba31467f2e611172334dc53a0e`
+const ATTESTATION_ISSUER = `0x503a5161D1c5D9d82BF35a4c80DA0C3Ad72d9244` // TODO: Update/ get from env post handoff
+
+const ATTESTATION_URL: Record<CHAIN_ID, string> = {
+  [CHAIN_ID.ETHEREUM]: 'https://easscan.org/graphql',
+  [CHAIN_ID.OPTIMISM]: 'https://optimism.easscan.org/graphql',
+  [CHAIN_ID.SEPOLIA]: 'https://sepolia.easscan.org/graphql',
+  [CHAIN_ID.OPTIMISM_SEPOLIA]: 'https://optimism-sepolia.easscan.org/graphql',
+  [CHAIN_ID.BASE]: 'https://base.easscan.org/graphql',
+  [CHAIN_ID.BASE_SEPOLIA]: 'https://base-sepolia.easscan.org/graphql',
+  [CHAIN_ID.ZORA]: '',
+  [CHAIN_ID.ZORA_SEPOLIA]: '',
+  [CHAIN_ID.FOUNDRY]: '',
+}
+
+export async function getDaoMultiSig(
```

Comment:
This file I am not totally understanding but thats only cause I dont know much about attestations. But looks like you're picking up the multisig address which I'm assuming is attested by the DAO ?